From 9fb72551715034ba7bad12a77dd8476bb289fab7 Mon Sep 17 00:00:00 2001
From: "Ooi, Joyce" <joyce.ooi@intel.com>
Date: Tue, 8 Oct 2019 01:48:59 -0700
Subject: [PATCH 017/120] arm64: dts: altera: update QSPI reg addresses for
 Stratix10

commit 6e041865ba0941c955e3a7f99d4dfada0d1afda9 from
https://github.com/altera-opensource/linux-socfpga.git

[backport 'Commit 0c33a70b3336 ("arm64: dts: altera: update QSPI
                                 reg addresses for Stratix10")']
This patch updates the reg addresses for QSPI boot and QSPI rootfs in
the device tree for Stratix10

Signed-off-by: Ooi, Joyce <joyce.ooi@intel.com>
Signed-off-by: Dinh Nguyen <dinguyen@kernel.org>
Integrated-by: Abhishek Paliwal <paliwal.abhishek@windriver.com>
---
 arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
index 66e769e31b95..645ad8009225 100644
--- a/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
+++ b/arch/arm64/boot/dts/altera/socfpga_stratix10_socdk.dts
@@ -178,12 +178,12 @@
 
 			qspi_boot: partition@0 {
 				label = "Boot and fpga data";
-				reg = <0x00910000 0x036F0000>;
+                                reg = <0x0 0x034B0000>;
 			};
 
 			qspi_rootfs: partition@4000000 {
 				label = "Root Filesystem - JFFS2";
-				reg = <0x4000000 0x4000000>;
+                                reg = <0x034B0000 0x0EB50000>;
 			};
 		};
 	};
-- 
2.31.1

